### PR TITLE
PR: feat: Notify users to manage asset on assethub, #4373

### PR DESCRIPTION
**Pull request by leocs2417**
issue link: https://github.com/opensquare-network/subsquare/issues/4373

### CODE REVIEW

File: packages/next-common/components/overview/accountInfo/components/assetHubManagePrompt.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import Link from "next/link";
+import { useCookieValue } from "next-common/utils/hooks/useCookieValue";
+import useAssetHubOnPolkadot from "../hook/useAssetHubOnPolkadot";
+import { AssetHubApiProvider } from "next-common/context/assetHub";
+import { AssetHubOnPolkadotMetadataProvider } from "next-common/components/overview/accountInfo/context/assetHubOnPolkadotMetadataContext";
+import Chains from "next-common/utils/consts/chains";
+import OnlyChain from "next-common/components/common/onlyChain";
+
+export function PromptContent() {
+  const [visible, setVisible] = useCookieValue(
```
Comment (hyifeng) [inline packages/next-common/components/overview/accountInfo/components/assetHubManagePrompt.js:14]: 这里和 <Prompt /> 内部的逻辑重复了

File: packages/next-common/components/overview/accountInfo/hook/useAssetHubOnPolkadot.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import BigNumber from "bignumber.js";
+import useRealAddress from "next-common/utils/hooks/useRealAddress";
+import { useEffect, useMemo, useState } from "react";
+import { useAssetHubOnPolkadotMetadata } from "../context/assetHubOnPolkadotMetadataContext";
+import { useAssetHubApi } from "next-common/context/assetHub";
+
+function useSubscribeMultiAssetAccounts(multiAccountKey) {
```
Comment (hyifeng) [inline packages/next-common/components/overview/accountInfo/hook/useAssetHubOnPolkadot.js:7]: 两个 useSubscribeMultiAssetAccounts 内容重复，可以提取公共逻辑

File: packages/next-common/components/overview/accountInfo/components/assetHubManagePrompt.js (hunk 1 above)
Comment (leocs2417) [reply packages/next-common/components/overview/accountInfo/components/assetHubManagePrompt.js]: done and thx.

File: packages/next-common/components/overview/accountInfo/hook/useAssetHubOnPolkadot.js (hunk 2 above)
Comment (leocs2417) [reply packages/next-common/components/overview/accountInfo/hook/useAssetHubOnPolkadot.js]: done and thx.

**Pull request merged**